jmix-flowui/flowui-themes/src/main/resources/META-INF/resources/themes/jmix-lumo/component/jmix-list-menu.css

```diff
@@ -44,19 +45,41 @@
   flex-shrink: 0;
 }
 
+.jmix-list-menu .jmix-menu-item-link .prefix-component,
+.jmix-list-menu .jmix-menubar-item .jmix-menubar-summary-icon-container .prefix-component {
+  flex-shrink: 0;
+}
+
+.jmix-list-menu .jmix-menu-item-link vaadin-icon.prefix-component,
+.jmix-list-menu .jmix-menubar-item .jmix-menubar-summary-icon-container vaadin-icon.prefix-component {
+  width: var(--lumo-icon-size-s);
+  height: var(--lumo-icon-size-s);
+}
+
 .jmix-list-menu .jmix-menubar-item .menubar-summary {
   white-space: nowrap;
   font-weight: bold;
   text-overflow: ellipsis;
   overflow: hidden;
 }
 
+.jmix-list-menu .jmix-menu-item-link .suffix-component,
+.jmix-list-menu .jmix-menubar-item .jmix-menubar-summary-icon-container .suffix-component {
+  margin-inline-start: auto;
+  flex-shrink: 0;
+}
+
+.jmix-list-menu .jmix-menubar-item .jmix-menubar-summary-icon-container > :last-child {
+  margin-right: var(--lumo-space-s);
```

Comment:
you can't use left/right variants, because they are not adapted to **rtl** mode. Use `inline-start/end` instead.

Please, fix all styles in this file, even existed